### DISCUSSION TITLE
Tiffslide-Xarray integration

### DISCUSSION THREAD

**Discussion by swamidass** [Bayer-Group/tiffslide#78] [Show and tell]
I've made a new plugin for Xarray that uses tiffslide to read tiff and svs files.

https://github.com/swamidasslab/tiffslide-xarray

```
pip install tiffslide-xarray
```

Our group is using this library to process whole-slide images in Pathology.  Let us know if this is helpful to you and how we can better design the API before fully stabilizing it.

**Comment by ap--** [comment]
Interesting! Thanks for posting it here 😃 
It's always nice to see tiffslide being used in other tools!

What kind of processing are you using xarray for when working with whole-slide images?

**Comment by swamidass** [comment]
See the discussion at #79 .

We've been using for handling WSI data for  deep learning, both during training and inference. See for example here:  https://pubmed.ncbi.nlm.nih.gov/29994669/
